resolve/matrix/Sparse.cpp

```diff
@@ -253,6 +362,14 @@ namespace ReSolve { namespace matrix {
     return 0;
   }
 
+  /**
+   * @brief updata matrix values using the _new_values_ provided either as HOST or as DEVICE array. This function only assigns a pointer, but does not copy. It sets update flags.
+   *
```

Comment:
Make brief really brief:
```suggestion
  /**
   * @brief updata matrix values using the _new_values_ provided either as HOST or as DEVICE array. 
   *
   * This function only assigns a pointer, but does not copy. It sets update flags.
   *
```